package.json

```diff
@@ -7,7 +7,7 @@
     "url": "https://github.com/percy/percy-storybook.git"
   },
   "engine": {
-    "node": ">=14"
+    "node": ">=16"
```

Comment:
we don't need to have this compulsion, our code will still work with node 14.

(though does it let us run just verify) ?

Reply:
Nope, storybook v7 is [**not** compatible with node 14](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#dropped-support-for-node-15-and-below).

![Screenshot 2023-10-19 at 9 49 29 PM](https://github.com/percy/percy-storybook/assets/22821480/11333fe4-b470-4fb7-945f-67380f48fc9b)